bridge-fungible-token-abi/src/lib.sw

```diff
@@ -1,17 +1,26 @@
 library bridge_fungible_token_abi;
```

Comment:
One thing we started doing in the predicates repo was avoid using names like 'main.sw' and 'lib.sw' and instead use names like 'contract_message_predicate.sw' or 'contract_message_receiver_abi.sw'

Reply:
I'll adjust accordingly.